includes/wf_crm_webform_postprocess.inc

```diff
@@ -1682,7 +1682,7 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
     //Fix IPN payments marked as paid by cheque
     if (empty($params['payment_instrument_id'])) {
       if ($params['payment_processor_id'] && $this->contribution_page['is_monetary']) {
-        $params['payment_instrument_id'] = 'Check';
+        $params['payment_instrument_id'] = key(CRM_Core_OptionGroup::values('payment_instrument', FALSE, FALSE, FALSE, 'AND is_default = 1'));
```

Comment:
The `key()` function takes it parameter by reference, so I'm not sure you can use it like this (at least not on all versions of php).